sc/src/main/javascript/bundles/gene-search/src/ExperimentTableCard.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
+import formatNumber from 'format-number'
 import EbiSpeciesIcon from 'react-ebi-species'
 
+const _formatNumber = formatNumber()
```

Comment:
Why do we need to define a `const` for `formatNumber()` instead of using it directly?

Reply:
`formatNumber()` returns a function which we then apply to each number. Otherwise we would need to do `formatNumber()(numberOfAssays)`, which incurs in an unnecessary (although negligible) overhead. I used it to be explicit about what `formatNumber` returns.